src/neo-vm/Debugger.cs

```diff
@@ -53,17 +53,19 @@ public VMState StepInto()
             if (engine.State.HasFlag(VMState.HALT) || engine.State.HasFlag(VMState.FAULT))
                 return engine.State;
             engine.ExecuteNext();
-            if (engine.State == VMState.NONE)
-                engine.State = VMState.BREAK;
```

Comment:
You can't remove these two lines.

The last line:

```c#
engine.ExecuteNext();
```

It is possible to cause the VM to enter the FAULT state.

Reply:
but fault is not none, i can't touch this line with unit test...

Reply:
I don't understand what do you mean?

Reply:
@erikzhang how can i enter in this condition?

Reply:
Call `StepInto()` when vm is in any non-fault state.